### PR TITLE
Fix ep10 ep11

### LINKED ISSUE #22: Code for Epizodes 10, 11 not yet published?
Even though the videos are already online, Matt, will you update the code to play with them? 

For ep.10, I get to choose a moving gif, and that's it. 
For 11 there's any empty window.

### PR DESCRIPTION
**Pull request by breznak**
Fixes #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +46,10 @@ Now start this server on any port you like:
     npm start 8001
 
 View on <http://localhost:8001/>.
+
+## Episodes
+
+Notes about the episodes. 
+
+- Ep. 10, ep. 11: need to generate the `webpack` static content from `cell-viz`
+- Ep. 10: You appear too close to the scene, press `s` to zoom-out. Currently works only in Chrome/Chromium browsers
```
Comment (rhyolight) [inline README.md:55]: You might also add:

- Ep. 1-6 do not require `nupic-history-server`

**Review by rhyolight — APPROVED** [review]

**Comment by rhyolight** [comment]
I see the gifdata bug you found there. Thanks for fixing.

**Comment by breznak** [comment]
Btw, what do you think about storing some version of `dyson-bundle.js` in the repository, to avoid users the hassle with rebuild in the other repo. And update it from time to time? I think that'd be the easiest

**Comment by rhyolight** [comment]
Yes that's fine, but put the release in the filename:

https://github.com/numenta/cell-viz/releases/tag/1.0.0

**Pull request merged**